.github/workflows/track_backlog.yml

```diff
@@ -0,0 +1,16 @@
+name: Track new issues in backlog
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "5 * * * *" # Run at 5 minutes past every hour
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Track all issues
+      uses: dhruvkb/issue-projector@0.0.1
+      with:
+        ACCESS_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+        ORG_NAME: 'creativecommons'
+        PROJECT_NAME: 'Backlog – CC Search'
```

Comment:
Can we use a project ID here so that if the name changes, it won't break? I plan to rename this to "Backlog", but in general, IDs are meant to be static, names aren't.

Reply:
That does make sense. Hate how GitHub distinguishes between IDs and "number"s, only allowing `GET` calls on the former but only showing the latter in the web UI.